apps/omg_watcher_rpc/lib/application.ex

```diff
@@ -19,7 +19,8 @@ defmodule OMG.WatcherRPC.Application do
 
   def start(_type, _args) do
     DeferredConfig.populate(:omg_watcher_rpc)
-
+    DeferredConfig.populate(:omg_child_chain_rpc)
+    :ok = Application.put_env(:spandex_phoenix, :tracer, OMG.WatcherRPC.Tracer, persistent: true)
```

Comment:
oh my, now I see, ouch. So `spandex_phoenix` is overwritten for every umbrella subapp, right?

If there isn't any other way to mend this, comments would be v. useful.

Reply:
will do!